src/databricks/labs/ucx/hive_metastore/tables.py

```diff
@@ -252,3 +267,24 @@ def _init_seen_tables(self):
 
     def _table_already_upgraded(self, target) -> bool:
         return target in self._seen_tables
+
+    def revert_migrated_tables(self, *, schema: str | None = None, table: str | None = None):
+        def scope_filter(cur_table: Table):
```

Comment:
Don't use inner function, it makes code harder to read and in this case doesn't bring any benefits

Reply:
Fixed